robots-ev3-module/src/main/java/orwell/tank/communication/UdpBroadcastDataDecoder.java

```diff
@@ -7,7 +7,7 @@
 import java.util.Arrays;
 
 public class UdpBroadcastDataDecoder {
-    private final static Logger logback = LoggerFactory.getLogger(UdpBroadcastDataDecoder.class);
+    private static final Logger logback = LoggerFactory.getLogger(UdpBroadcastDataDecoder.class);
     private static final String PROXY_IP_FIELD_NAME = Character.toString('*');
     private boolean isPacketDataCorrect;
```

Comment:
This should probably start as false.

Reply:
Okay. Java default value initialization is false, but if you ask, it's better to explicit it.

Reply:
I do not Java enough.